wasm/lib/src/objjsvalue.rs

```diff
@@ -0,0 +1,154 @@
+use js_sys::{Function, JsString, Reflect};
+use wasm_bindgen::JsValue;
+
+use rustpython_vm::function::{Args, OptionalArg};
+use rustpython_vm::obj::objstr::PyStringRef;
+use rustpython_vm::pyobject::{PyContext, PyObjectRef, PyRef, PyResult, PyValue};
+use rustpython_vm::VirtualMachine;
+
+use crate::convert;
+
+fn get_prop(value: JsValue, name: &str, vm: &VirtualMachine) -> Option<PyObjectRef> {
+    let name: &JsString = &name.into();
+    if Reflect::has(&value, name).expect("Reflect.has failed") {
+        Some(convert::js_to_py_with_this(
+            vm,
+            Reflect::get(&value, name).expect("Reflect.get failed"),
+            Some(value),
+        ))
+    } else {
+        None
+    }
+}
+
+fn set_prop(value: &JsValue, name: &str, val: PyObjectRef, vm: &VirtualMachine) {
+    Reflect::set(value, &name.into(), &convert::py_to_js(vm, val)).expect("Reflect failed");
+}
+
+#[pyclass(name = "JsValue")]
```

Comment:
I mentioned [here](https://github.com/RustPython/RustPython/issues/808#issuecomment-485042199) also, but I think this should actually be called `JsObject`, with primitives always be converted to their Python correspondents.

Also, I think the name exposed to Python for this and other JS types should be `JS` (fully capitalized), as that's the convention set by [PEP 8](https://www.python.org/dev/peps/pep-0008/).

> Note: When using acronyms in CapWords, capitalize all the letters of the acronym. Thus HTTPServerError is better than HttpServerError.